app/views/users/invitations/error.js.erb

```diff
@@ -0,0 +1 @@
+$("#error_msg").html("<%= j render("users/shared/error_messages", resource: resource) %>")
```

Comment:
@ajaystha42  again --  this file does the same thing as app/views/users/invitations/update.js.erb -- create a partial and reuse it